packages/api/lambdas/sf-event-sqs-to-db-records/write-execution.js

```diff
@@ -130,6 +130,13 @@ const writeExecutionToDynamoAndES = async (params) => {
   }
 };
 
+const publishExecutionSnsMessage = async (record) => {
+  const topicArn = process.env.execution_sns_topic_arn;
```

Comment:
We should use `@cumulus/common/env.getRequiredEnvVar` to get this env var, because it will throw a specific error if the environment variable is missing, which is more helpful than the confusing error we might see from SNS if we try to publish when this variable is undefined